Vagrantfile

```diff
@@ -10,11 +10,12 @@ Vagrant.configure(2) do |config|
   config.vm.define "elasticsearch" do |elasticsearch|
     elasticsearch.vm.box = "debian/jessie64"
     elasticsearch.vm.provider "virtualbox" do |vb|
-      vb.memory = "2048"
+      vb.memory = "4096"
```

Comment:
Kept on running into issues with the plugins and the jvm not having enough memory that bumping the value fixed it. I know it's not ideal, but guess if ES 5.x is allocating 2GB by default now, might be best to have the VM at 4.